UnitySDK/Assets/ML-Agents/Plugins/Barracuda.Core/Barracuda/Resources/BarracudaReferenceImpl.compute

```diff
@@ -0,0 +1,863 @@
+﻿#pragma kernel Dense
```

Comment:
Are you shipping with both the DLL and the source?

Reply:
The compute files cannot be put in a DLL.